wit/deps/cli/imports.wit

```diff
@@ -0,0 +1,20 @@
+package wasi:cli@0.2.0;
+
+world imports {
+  include wasi:clocks/imports@0.2.0;
+  include wasi:filesystem/imports@0.2.0;
+  include wasi:sockets/imports@0.2.0;
+  include wasi:random/imports@0.2.0;
+  include wasi:io/imports@0.2.0;
+
```

Comment:
Curious to know the reason why you needed to manually import these here

Reply:
Right now, mdk_rs doesn't have a functioning logging and I wanted to add that (the stderr worlds). On the host side, the lib.rs/wasmtime_wasi implements this directly without adding the wit files exactly. But on the guest side, there's only lib.rs/wasi available which exposes all worlds at once. Since we only need logging, might as well selectively add the worlds we need.

Also, the `componentize-py` resulting components depend on all of the command worlds.